website/static/js/contribAdder.js

```diff
@@ -158,15 +158,22 @@ var AddContributorViewModel = oop.extend(Paginator, {
     getContributors: function() {
         var self = this;
         self.notification(false);
-        return $.getJSON(
-            self.nodeApiUrl + 'get_contributors/', {},
-            function(result) {
-                var contributors = result.contributors.map(function(userData) {
-                    return userData.id;
-                });
-                self.contributors(contributors);
-            }
-        );
+        var url = window.contextVars.apiV2Prefix + 'nodes/' + window.contextVars.node.id + '/contributors/';
+
+        return $.ajax({
```

Comment:
There's an [osfHelper function](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/osfHelpers.js#L65) that would simplify this request, but it's not strictly necessary